admin_app/src/app/content/page.tsx

```diff
@@ -128,13 +179,22 @@ const CardsGrid = ({
     apiCalls
       .getContentList({ token: token!, skip: 0, limit: MAX_CARDS_TO_FETCH })
       .then((data) => {
-        const filteredData = data.filter(
-          (card: Content) =>
+        const filteredData = data.filter((card: Content) => {
+          const matchesSearchTerm =
             card.content_title
               .toLowerCase()
               .includes(searchTerm.toLowerCase()) ||
-            card.content_text.toLowerCase().includes(searchTerm.toLowerCase()),
-        );
+            card.content_text.toLowerCase().includes(searchTerm.toLowerCase());
+
+          const matchesAllTags = filterTags.every((fTag) =>
+            card.content_tags.includes(fTag.tag_id)
+          );
```

Comment:
I  see that the filtering seems to be an AND, let's do an OR instead as you said? To me showing all contents that have either tag selected in the filter box makes more intutive sense

Reply:
I think we agreed on AND as far as I can remember. Let's discuss on Slack